src/java/apiview-java-processor/result/final-1.0.2-sources.json

```diff
@@ -0,0 +1,4659 @@
+{
```

Comment:
The files in `result` directory can be deleted in this PR

Reply:
Removed